git_pull/README.md

```diff
@@ -0,0 +1,121 @@
+# Hass.io Core Add-on: Git pull
+
+Load and update configuration files for Home Assistant from a Git repository.
+
+![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+
+## About
+
+You can use this add-on to `git pull` updates to your Home Assistant configuration files from a Git
+repository.
+
+## Installation
+
+The installation of this add-on is straightforward and easy to do.
+
+1. Navigate in your Home Assistant frontend to **Hass.io** -> **Add-on Store**.
+2. Find the "Git pull" add-on and click it.
+3. Click on the "INSTALL" button.
+
+## How to use
+
+In the configuration section, set the repository field to your repository's
+clone URL and check if any other fields need to be customized to work with
+your repository. Next,
+
+1. Start the add-on.
+2. Check the add-on log output to see the result.
+
+If the log doesn't end with an error, the add-on has successfully
+accessed your git repository. Examples of logs you might see if
+there were no errors are: `[Info] Nothing has changed.`,
+`[Info] Something has changed, checking Home-Assistant config...`,
+or `[Info] Local configuration has changed. Restart required.`.
+
+If you made it this far you may want to let the add-on automatically
+check for updates by setting the `active` field (a subfield of `repeat`)
+to `true` and turning on "Start on boot."
+
+## Configuration
+
+Add-on configuration:
+
+```json
+{
+  "git_branch": "master",
+  "git_command": "pull",
+  "git_remote": "origin",
+  "git_prune": "false",
+  "repository": "https://example.com/my_configs.git",
+  "auto_restart": false,
+  "restart_ignore": [
+    "ui-lovelace.yaml",
+    ".gitignore",
+    "exampledirectory/"
+  ],
+  "repeat": {
+    "active": false,
+    "interval": 300
+  },
+  "deployment_user": "",
+  "deployment_password": "",
+  "deployment_key": [
+"-----BEGIN RSA PRIVATE KEY-----",
+"MIIEowIBAAKCAQEAv3hUrCvqGZKpXQ5ofxTOuH6pYSOZDsCqPqmaGBdUzBFgauQM",
+"xDEcoODGHIsWd7t9meAFqUtKXndeiKjfP0MMKsttnDohL1kb9mRvHre4VUqMsT5F",
+"...",
+"i3RUtnIHxGi1NqknIY56Hwa3id2yk7cEzvQGAAko/t6PCbe20AfmSQczs7wDNtBD",
+"HgXRyIqIXHYk2+5w+N2eunURIBqCI9uWYK/r81TMR6V84R+XhtvM",
+"-----END RSA PRIVATE KEY-----"
+  ],
+  "deployment_key_protocol": "rsa"
+}
+```
+
+### Options
+
+- **git_remote** (*Required*): Name of the tracked repository. Leave this as `origin` if you are unsure.
```

Comment:
Please check the README's of the other add-ons for the formatting of the options.
It would be nice if we could stay consistent.

Reply:
Updated. Let me know what you think.